simapp/ante.go

```diff
@@ -32,6 +32,10 @@ func NewAnteHandler(options HandlerOptions) (sdk.AnteHandler, error) {
 		return nil, errors.New("sign mode handler is required for ante builder")
 	}
 
+	if options.UnorderedTxManager == nil {
+		return nil, errors.New("unordered tx manager is required for ante builder")
+	}
```

Comment:
So in this case we are requiring you to have an unorderedTx manager.  Which is not how we described the functionality in the docs.  Can this function reflect the fact that `UnorderedTx` is optional and completely not required?

I just see that folks will copy this file from `simapp` for what their ante chain should be.

```
	if options.UnorderedTxManager == nil {
		// add in unordered tx manager
	}
```

Reply:
[optional utx](https://github.com/cosmos/cosmos-sdk/pull/23708/commits/1b6d687309ba23c804ba3caecdab599c5a7d4132)